### PR TITLE
Example of Hibernate OGM with Spring Data JPA

### PR DESCRIPTION
**Pull request by DavideD**
A simple example about using Hibernate OGM with Spring Data

### CODE REVIEW

File: hibernate-ogm/spring-data-jpa/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: java
+jdk:
+  - oraclejdk7
+  - oraclejdk8
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/.travis.yml]: I don't think a travis file is useful here? Especially if it's empty ;)

File: hibernate-ogm/spring-data-jpa/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Building the project
+
+Execute the following command to build the project:
+
+    mvn clean install
+
+This will execute some simple JUnit tests which use JPA's RESOURCE_LOCAL mode to access the datastore.
+
+Any feedback on this demo is highly welcome, just send a mail to the hibernate-dev mailing list.
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/README.md]: Include a link to the subscription page for the hibernate-dev list (https://lists.jboss.org/mailman/listinfo/hibernate-dev).

File: hibernate-ogm/spring-data-jpa/pom.xml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    <name>Hibernate OGM and Spring Data JPA</name>
+    <description>Simple example of HIbernate OGM working with Spring Data JPA</description>
+
+    <!-- Properties -->
+    <properties>
+        <!-- Dependencies -->
+        <commons.lang3.version>3.4</commons.lang3.version>
+        <h2.version>1.4.192</h2.version>
+        <hibernate.version>5.1.0.Final</hibernate.version>
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/pom.xml]: Please use the latest point release.

File: hibernate-ogm/spring-data-jpa/pom.xml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        <jta.version>1.1</jta.version>
+        <junit.version>4.12</junit.version>
+        <logback.version>1.1.7</logback.version>
+        <slf4j.version>1.7.21</slf4j.version>
+        <spring.version>4.2.6.RELEASE</spring.version>
+        <spring.data.version>1.10.1.RELEASE</spring.data.version>
+
+        <!-- Plugins -->
+        <versioneye.maven.version>3.9.2</versioneye.maven.version>
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/pom.xml]: Looks like it's not used?

File: hibernate-ogm/spring-data-jpa/pom.xml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.hibernate.ogm</groupId>
+            <artifactId>hibernate-ogm-core</artifactId>
+            <version>5.1.0.Final</version>
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/pom.xml]: Make it a property.

File: hibernate-ogm/spring-data-jpa/pom.xml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.hibernate.ogm</groupId>
+            <artifactId>hibernate-ogm-neo4j</artifactId>
+            <version>5.1.0.Final</version>
```
Comment (gsmet) [inline hibernate-ogm/spring-data-jpa/pom.xml]: Same here.

**Comment by gsmet** [comment]
Hmmm, where is Hibernate OGM used? I see a Spring Data example but I don't see how it interacts with OGM. Did I miss something?

**Comment by DavideD** [comment]
I'll update it and send a new pr

**Pull request closed without merging**